lib/howitzer/meta/element.rb

```diff
@@ -0,0 +1,34 @@
+module Howitzer
+  module Meta
+    # This class represents element entity within howitzer meta information interface
+    class Element
+      attr_reader :name, :context
+
+      include Howitzer::Meta::Actions
+      # Creates new meta element with meta information and utility actions
+      # @param name [String] name of the element
+      # @param context [Howitzer::Web::Page] page element belongs to
+      def initialize(name, context)
+        @name = name
+        @context = context
+      end
+
+      # Finds all instances of element on the page and returns them as array of capybara elements
+      # @param *args [Array] arguments for elements described with lambda locators
+      # @return [Array]
+      def capybara_elements(*args)
+        context.send("#{name}_elements", *args)
+      end
+
+      # Finds element on the page and returns as a capybara element
+      # @param *args [Array] arguments for elements described with lambda locators
+      # @param wait [Integer] wait time for element search
+      # @return [Capybara::Node::Element, nil]
+      def capybara_element(*args, wait: 0)
+        context.send("#{name}_element", *args, match: :first, wait: wait)
```

Comment:
Unfortunately it will not work properly

```ruby
2.1.6 :001 > def foo(*args)
2.1.6 :002?>   puts args.inspect
2.1.6 :003?>   end
 => :foo 
2.1.6 :004 > def bar(*args)
2.1.6 :005?>   foo(*args, match: :first)
2.1.6 :006?>   end
 => :bar 
2.1.6 :007 > bar(1, a: 2)
[1, {:a=>2}, {:match=>:first}]
 => nil 
```

Reply:
hash arguments should be merged with `match: :first`. You should see how it is implemented in https://github.com/strongqa/howitzer/blob/master/lib/howitzer/web/element_dsl.rb#L24

Reply:
**Please cover it with tests!!!**